examples/go-xdp-counter/main.go

```diff
@@ -36,7 +36,7 @@ const (
 	XDP_ACT_OK = 2
 )
 
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc clang -no-strip -cflags "-O2 -g -Wall" bpf ./bpf/xdp_counter.c -- -I.:/usr/include/bpf:/usr/include/linux
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc clang -no-strip -cflags "-O2 -g -Wall" -target amd64,arm64,ppc64le,s390x bpf ./bpf/xdp_counter.c -- -I.:/usr/include/bpf:/usr/include/linux
```

Comment:
In each of the example main.go files, `DefaultByteCodeFile` (line 27 in this file) needs to be changed to `bpf_x86_bpfel.o`.

Also, write an issue, because now the examples will only work on x86, unless the file is passed in. I thought bpfman was going to auto-detect, but will need that functionality in examples common code.

Reply:
Good catch, I just haven't been focused on the locally present byte-code images at all :/ It wouldn't be hard for us to have a `--cilium_ebpf_project` flag for `bpfman load file` though if that's the direction you'd rather go!

Reply:
I had something coded up when I first started multi-arch, but since your were handling bytecode, I abandoned it. Just write an issue and I'll figure it out later.
See: https://github.com/bpfman/bpfman/pull/1139

Reply:
https://github.com/bpfman/bpfman/issues/1190